Analytics/importers/tfl_bike_points.py

```diff
@@ -0,0 +1,41 @@
+import os, sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+from importers.base import BaseImporter, Location, get_config
+from models.sensor import Sensor
+from models import location
+import pandas as pd
+import numpy as np
+
+import yaml
```

Comment:
Accepting for now but this shouldn't be needed. We need to address the method by which we source the config file as it's probably not best-practice at the moment.